share/nisar/defaults/gcov.yaml

```
@@ -570,14 +570,27 @@ runconfig:
                 # Default: True
                 qa_reports: true
 
+            software_config:
+
+                # True to use memory map(s) to cache select Dataset(s).
```

```suggestion
                # True to use memory map(s) to cache select dataset(s).
```
Is there a specific reason "Dataset" is capitalized here? If not, we should lowercase it.

---

For HDF5 Datasets, we've been using a capital "D".

---

We define that in the product specs document, but here it's outside that context and appears to be a grammatical error.